app/src/main/res/layout/fragment_delete_browsing_data.xml

```diff
@@ -51,17 +51,6 @@
                 android:focusable="true"
                 app:deleteBrowsingDataItemTitle="@string/preferences_delete_browsing_data_browsing_data_title"
                 app:deleteBrowsingDataItemSubtitle="@string/preferences_delete_browsing_data_browsing_data_subtitle" />
-            <org.mozilla.fenix.settings.deletebrowsingdata.DeleteBrowsingDataItem
-                android:id="@+id/collections_item"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="8dp"
-                android:layout_marginBottom="8dp"
-                android:background="?android:attr/selectableItemBackground"
-                android:clickable="true"
-                android:focusable="true"
-                app:deleteBrowsingDataItemTitle="@string/preferences_delete_browsing_data_collections_title"
```

Comment:
If these strings are unused, can you also delete them from the strings.xml file? no need to do it for every language, just the top-level one is fine.

Reply:
Done! I have removed the strings form all the files and committed the latest changes. Please review.